string operation
string operation_data_1
float32 operation_data_2
---
string track_title
float32 track_elapsed_time
float32 track_total_time
bool status
